pipelines/fbc-release/README.md

```diff
@@ -19,8 +19,12 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                    | Yes       | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                           | No        | -                                                               |
 
+
+## Changes in 4.0.4
+* The `sign-index-image` task now receives new mandatory parameter `releasePlanAdmissionPath`
+
 ## Changes in 4.0.3
-* The `sign-index-image` task now receives new mandatory parameters `taskGitUrl` and `taskGitRevision
+* The `sign-index-image` task now receives new mandatory parameters `taskGitUrl` and `taskGitRevision'
```

Comment:
Good catch, but actually wrong ;) Shouldn't be `'` but backtick.

Reply:
fixed